blueprint/src/chapter/AdeleMiniproject.tex

```diff
@@ -946,6 +947,8 @@ \subsection{Base change for adeles}
 
 \begin{theorem}
   \label{NumberField.AdeleRing.baseChange_moduleTopology}
+  \lean{NumberField.AdeleRing.instBaseChangeIsModuleTopology}
+  \leanok
```

Comment:
`\leanok` in the statement means "the statement of the theorem is formalized". Can you add it to the proof as well?

I am currently going through the Haar character project and getting all the LaTeX up to date. When this is done I'll start on the adele project.